src/OrchardCore.Modules/OrchardCore.Resources/LocalResourceConfigureOptions.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.ResourceManagement.Core;
+
+namespace OrchardCore.Resources;
+
+public class LocalResourceConfigureOptions : IConfigureOptions<ResourceSetting>
+{
+    private readonly IConfiguration _configuration;
+
+    public LocalResourceConfigureOptions(IConfiguration configuration)
+    {
+        _configuration = configuration;
+    }
+
+    public void Configure(ResourceSetting options)
+    {
+        options = _configuration.GetSectionCompat("OrchardCore:Site_Settings").Get<ResourceSetting>();
```

Comment:
Use bind instead of get as - get creates new instance

Reply:
done! good call